views/record_timestamp.go

```diff
@@ -1,30 +1,56 @@
 package views
 
 import (
-	"fmt"
+	"strings"
 	"time"
 
 	"github.com/ugent-library/biblio-backoffice/ctx"
 	"github.com/ugent-library/biblio-backoffice/models"
 )
 
 func CreatedBy(c *ctx.Ctx, createdAt *time.Time, createdBy *models.Person) string {
-	s := fmt.Sprintf("Created %s", createdAt.In(c.Timezone).Format("2006-01-02 15:04"))
+	sb := strings.Builder{}
```

Comment:
better to store a pointer with `&strings.Builder{}`, all methods of builder are pointer methods: https://pkg.go.dev/strings#Builder.WriteString

Reply:
Fixed in 176b383ba71f58c9db7240d5e809c84700e6635d